src/workloads/replication/startup/1_1_5GB_crud.yml

```diff
@@ -0,0 +1,70 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: >
+  Adds CRUD operations to be replayed during startup recovery for the light phase.
+  To know more about the test phases please refer to 'src/workloads/replication/startup/README.md'.
+
+  Expected behavior:
+  --------------
+  We expect after restarting mongod after running this workload to have alot of CRUD ops to be
+  applied during startup recovery:
+
+  Sample logs:
+  +--------------------------------------------------------+
+  |  {                                                     |
+  |       s: "I",                                          |
+  |       c: "REPL",                                       |
+  |       id: 21536,                                       |
+  |       ctx: "initandlisten",                            |
+  |       msg: "Completed oplog application for recovery", |
+  |       attr: {                                          |
+  |           numOpsApplied: 50207,                        |
+  |           numBatches: 142,                             |
+  |           applyThroughOpTime: {                        |
+  |               ts: {                                    |
+  |                   $timestamp: {                        |
+  |                       t: 1690277528,                   |
+  |                       i: 1                             |
+  |                   }                                    |
+  |               },                                       |
+  |               t: 6                                     |
+  |           }                                            |
+  |       }                                                |
+  |   }                                                    |
+  +--------------------------------------------------------+
+
+Keywords:
+- startup
+- stopCheckpointing
+- updates
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2500
+
+NumPhases: &max_phase 1
```

Comment:
nit: could we add these params into a named block?

(and in the rest of the files also)

Reply:
Done